.scrutinizer.yml

```diff
@@ -20,7 +20,12 @@ tools:
 build_failure_conditions:
     - 'issues.label("coding-style").exists'
 
+checks:
+  php:
+    duplication: false
+
 filter:
   excluded_paths:
     - docs/*
     - vendor/*
+    - src/PhpSpec/Formatter/Presenter/StringPresenter.php
```

Comment:
This is part of a temp commit that was introduced to show the improvement the refactoring has on the code quality metrics. It should be removed before merging.